api/errors.js

```diff
@@ -10,7 +10,10 @@ exports.handlePSQLErrors = (err, req, res, next) => {
   if (err.code === "22P02" || err.code === "42703") {
     res.status(400).send({ msg: "bad request" });
   } else if (err.code === "23503") {
-    res.status(404).send({ msg: "not found" });
+    
+    const additionalDetail = err.constraint === 'comments_author_fkey' ? "user" : "article";
+
+    res.status(404).send({ msg: `${additionalDetail} not found` });
```

Comment:
This is an interesting way of checking which column is missing in your table using the additional information PSQL send you. There's more on those errors than just codes for you to use, so I've got no problem with how you've built this

Reply:
Cool, ta!  I'll merge now :)